app/js/deleteQuestion.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * soft deletes question from database
+ * @param {number} id the question id
+ * @returns {Promise<*>} response from api
+ */
+
```

Comment:
Inconsistent with white space theme for documentation. Normally there isn't a gap.